scripts/launch_stampy.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
 
-SCRIPT=$(basename $0);
+set -e
```

Comment:
Yes. It ensures if one system call fails, the script exits and doesn't waste time looking for output that wasn't properly made due to an error.